### ISSUE TITLE
large custom structure type gets rendered wrong in view and edit modes, referenced wrong

### ISSUE THREAD

**Issue by droogie** [Vector35/binaryninja-api#3892]
**Version and Platform (required):**
- Binary Ninja Version: 3.3.4061-dev (Build ID 2066fa08)
- OS: Arch Linux
- OS Version: N/A
- CPU Architecture: x64

**Bug Description:**
When creating a large custom type it will display char arrays as `__padding _XX[YY]` in the standard display window. If you open in an editor those arrays will be duplicates, code referencing data within those offsets seem to not reference correctly.

**Steps To Reproduce:**
Please provide all steps required to reproduce the behavior:
1. Create a new type by pressing `i` and entering the following:
```
struct new_obj __packed {
};
```
2. Click on the `new_obj` type and press `l` for a new size, enter `0x8c8` (should also be currently set to packed with size of 1)
3. assign the type to a function signature
4. press `y` on the `new_obj` type to edit and you'll see the duplicated array, for example:
```
struct new_obj __packed
{
    __padding char _0[8];
    __padding char _8[8];
    __padding char _10[8];
    __padding char _18[8];
    __padding char _20[8];
    __padding char _28[0x8a0];
    __padding char _28[0x8a0];
};
```

**Expected Behavior:**
I don't expect to see the `__padding` keyword in the regular view. The array should not be duplicated and references to offsets within the array should be named `__offset(XX).Y`.

**Screenshots:**
![screen_shot_2023-02-13_11:56-04-706866986](https://user-images.githubusercontent.com/1254882/218561549-d968643c-8f9c-473a-aa42-7f56f5a69ccc.png)
![screen_shot_2023-02-13_11:56-11-683553273](https://user-images.githubusercontent.com/1254882/218561576-9a2e34aa-eb55-4925-a551-6ba0a7a0d838.png)

**Comment by CouleeApps** [comment]
The extra `__padding` in the types window was added to prevent trying to render 10000s of lines of ?? ?? ?? ?? data for large structs (see #3761 and #3600). It seems like it has crept into the struct definition editor as well. Those lines need to be removed when the structure is rendered to text for the type editor.

**Comment by CouleeApps** [comment]
This will be fixed in 3.4.4108-dev. Just needed to not render the padding line twice.
